### PR TITLE
chore(api)!: updating api endpoints

### PR DESCRIPTION
**Pull request by gphorvath**

### CODE REVIEW

File: src/leapfrogai_api/routers/leapfrogai/auth.py [hunk 1]
```diff
@@ -82,7 +82,7 @@ async def list_api_keys(
     return await crud_api_key.list()
 
 
-@router.post("/update-api-key/{api_key_id}")
+@router.post("/api-keys/{api_key_id}")
```
Comment (andrewrisse) [inline src/leapfrogai_api/routers/leapfrogai/auth.py:85]: recommend change to PATCH

Comment (gphorvath) [reply src/leapfrogai_api/routers/leapfrogai/auth.py]: changed this in the API

**Review by andrewrisse — DISMISSED** [review]

**Comment by jalling97** [comment]
Since we haven't pulled the API integration tests into CI yet, are these changes passing those tests?

File: src/leapfrogai_api/Makefile [hunk 2]
```diff
@@ -36,4 +40,4 @@ env:
 	$(call get_jwt_token,"${SUPABASE_URL}/auth/v1/token?grant_type=password")
 
 test-integration:
-	cd ../../ && python -m pytest tests/integration/api/ -vv -s
+	@cd ${MAKEFILE_DIR} && python -m pytest ../../tests/integration/api/test_auth.py -vv -s
```
Comment (CollectiveUnicorn) [inline src/leapfrogai_api/Makefile:43]: Is limiting the scope of these tests intentional?

File: .github/workflows/e2e.yaml [hunk 3]
```diff
@@ -129,9 +129,11 @@ jobs:
 
         # Run the playwright UI tests using the deployed Supabase endpoint
         - name: UI/API/Supabase E2E Playwright Tests
-          run: |
-            cp src/leapfrogai_ui/.env.example src/leapfrogai_ui/.env
-            TEST_ENV=CI PUBLIC_DISABLE_KEYCLOAK=true PUBLIC_SUPABASE_ANON_KEY=$ANON_KEY npm --prefix src/leapfrogai_ui run test:integration:ci
+          run: | 
+            echo "skip"
+#          run: |
```
Comment (CollectiveUnicorn) [inline .github/workflows/e2e.yaml:134]: Are these not working?

**Review by CollectiveUnicorn — CHANGES_REQUESTED** [review]

File: .github/workflows/e2e.yaml (hunk 3 above)
Comment (jalling97) [reply .github/workflows/e2e.yaml:134]: At the moment yes. @andrewrisse and I are looking into it but it appears to be a playwright issue, not that the UI or backend is broken at the moment.

File: src/leapfrogai_api/Makefile (hunk 2 above)
Comment (gphorvath) [reply src/leapfrogai_api/Makefile:43]: no... good catch!

Comment (gphorvath) [reply src/leapfrogai_api/Makefile]: fixed!

**Review by CollectiveUnicorn — APPROVED** [review]

**Pull request merged**